runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -2564,14 +3246,28 @@ TR::Node *TR_VectorAPIExpansion::transformNary(TR_VectorAPIExpansion *opt, TR::T
 
       for (int32_t i = 0; i < numOperands; i++)
          {
-         if (operands[i]->getOpCodeValue() == TR::aload)
+         TR::Node* operand = operands[i];
```

Comment:
The majority in this file use the style `TR::Node *operand` instead of `TR::Node* operand`.

Reply:
Done